docs/TerraformPlanResultsPR.md

```diff
@@ -0,0 +1,47 @@
+## [TerraformPlanResultsPR](../src/TerraformPlanResultsPR.groovy)
+
+Use this to post Terraform plan results in the comments of a PR.
+
+One-Time Setup:
+* Install the terraform-landscape gem on your Jenkins slaves. (ONLY when using `withLandscape(true)`)
```

Comment:
Can we please change "Jenkins slaves" to "Jenkins agents" or "Jenkins build nodes"?

Reply:
yes, changed it